### ISSUE TITLE
Не работает библиотека

### ISSUE THREAD

**Issue by melnikove** [UsefulWeb/AmoCRM#2]
Добрый день! 
Я пытаюсь испольщоваеть вашу библиотеку в реакт-приложении+вебпак.
Вебпак ругается на static properties:
`ERROR in ./node_modules/amocrm-js/src/base/ResourceFactoryBuilder.js
Module parse failed: Unexpected token (4:19)
You may need an appropriate loader to handle this file type.
| 
| class ResourceFactoryBuilder {
|   static factories = factories;
| 
|   constructor( request ) {
 @ ./node_modules/amocrm-js/src/AmoCRM.js 4:0-67 20:24-46
 @ ./src/containers/AdminProductMenu.js
 @ ./src/index.js
 @ multi (webpack)-dev-server/client?http://localhost:8090 webpack/hot/dev-server babel-polyfill react-hot-loader/patch ./src/index

ERROR in ./node_modules/amocrm-js/src/base/requests/domain/PrivateDomainRequest.js
Module parse failed: Unexpected token (7:30)
You may need an appropriate loader to handle this file type.
| 
| class PrivateDomainRequest extends DomainRequest {
|   static responseHandlerClass = PrivateDomainResponseHandler;
|   static NETWORK_PROTOCOL = 'https';
| 
 @ ./node_modules/amocrm-js/src/AmoCRM.js 3:0-79 13:24-44
 @ ./src/containers/AdminProductMenu.js
 @ ./src/index.js
 @ multi (webpack)-dev-server/client?http://localhost:8090 webpack/hot/dev-server babel-polyfill react-hot-loader/patch ./src/index
`
Я поставил плагин transform-class-properties, и прописал его в .babelrc, как указано в документации:
`{
  "presets": [
    [
      "latest", {
        "es2015": {
          "modules": false
        }
      }
    ],
    "react",
  ],
  "plugins": [
    "react-hot-loader/babel",
    ["transform-class-properties", { "loose": true }]
  ]
}`
Результат в итоге никак не изменился, также ругается. Указывал плагин и без параметра { "loose": true }, результат тот же. Пробовал использовать babel-preset-stage-2, также без результата.

Напишите, пожалуйста, в чем может быть проблема?

**Comment by neizerth** [comment]
@melnikove Добавил транспиляцию через babel со stage-0. Сейчас должно работать и с более ранними стадиями.
Прошу прощения за неудобства!
Всё ли сейчас в порядке?

**Comment by melnikove** [comment]
Спасибо, я как раз тоже разобрался со stage-2, и сейчас static-свойства отрабатывает)
Возник другой вопрос - webpack пишет,что не разрешает для библиотек unirest, mime, request, forever-agent, tough-cookie, tough-cookie использовать библиотеки fs, tls и net. Насколько я понимаю, это библиотеки для node.js. Т.е. Вашу библиотеку нельзя использовать для доступа фронтенда через API к AmoCRM? Только на стороне сервера?

**Comment by neizerth** [comment]
Да, и я бы не рекомендовал использовать ее на клиенте, так как вы будете передавать в конструктор конфиденциальные данные, по которым можно управлять порталом Amo. Это значит, ваши клиенты могут безвозвратно потерять сделки, Контакты и так далее.
Куда лучше сделать надстройку для запросов на сервер.

**Comment by melnikove** [comment]
Ок,  спасибо!
